Apps/Inc/main.c

```diff
@@ -0,0 +1,14 @@
+/* Copyright (c) 2020 UT Longhorn Racing Solar */
+
+#include "BSP_CAN.h"
+#include "config.h"
+// #include "ALEX FOLDER " ==> will hold checkVoltage() function
+
+int main(void){
+    while(checkVolatge() > 0){
+        BSP_CAN_Write(0x10B, checkVoltage());     //send message to CAN Controller 
```

Comment:
Can you change `0x10b` to a `#define`? I think it will make the code more readable

Reply:
so I change the top to say #define "0x1b" ? then leave the 0x10b there ?

Reply:
#define ALLCAPSVARIABLE 0x1b